Review new event tree

fancyElise opened the pull request:
At Last

It's messy, but it's done.

fancyElise committed:
latest changes to VComponent

fancyElise committed:
eventTree continuing work

fancyElise committed:
homygosh it MOSTLY passes

fancyElise committed:
EventTree with passing tests; still needs Test Case doc, but tests pass and cover.
Incorporated a stopReplication function in VComponent that all inheritors must implement.

ceodenwald commented on VASTTesting/AV.cpp:
```diff
@@ -21,6 +21,11 @@ void AV::update(timestamp t, dataMap dataMap)
 
 }
 
+void AV::stopReplication(bool another, string runID)
```
Can you document what might need to go into this function?

ceodenwald commented on VASTTesting/Environment.cpp:
```diff
@@ -21,6 +21,11 @@ void Environment::update(timestamp t, dataMap dataMap)
 
 }
 
+void Environment::stopReplication(bool another, string runID)
```
Same for this function.... what is expected to be in this function

ceodenwald commented on VASTTesting/Obstacle.cpp:
```diff
@@ -22,6 +22,11 @@ void Obstacle::update(timestamp time, dataMap updateMap)
 
 }
 
+void Obstacle::stopReplication(bool another, string runID)
```
Document this function

ceodenwald reviewed (changes requested):
I left comments on missing documentation.

fancyElise committed:
updated doc with regards to review comments;
renamed main project;
deleted old empty project

everything builds & tests all pass - make sure to clean, rebuild!
